ibm/service/vpc/data_source_ibm_is_backup_policy_job.go

```diff
@@ -0,0 +1,466 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
+
+package vpc
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/IBM-Cloud/terraform-provider-ibm/ibm/flex"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+	"github.com/IBM/vpc-go-sdk/vpcv1"
+)
+
+func DataSourceIBMIsBackupPolicyJob() *schema.Resource {
+	return &schema.Resource{
+		ReadContext: dataSourceIBMIsBackupPolicyJobRead,
+
+		Schema: map[string]*schema.Schema{
+			"backup_policy_id": &schema.Schema{
```

Comment:
I think in all VPC resource/datasource we did not used explictly "_id" to refer id 
ex vpc, subnet, ...can we follow same pattern

Reply:
backup_policy_id is used same in backup policy plan and in other resource hence we can retain this in order to maintain the consistency